_eap-quickstarts

```diff
@@ -1 +1 @@
-Subproject commit 8d9b0b373f688a1d04ad50fea78e349721f4da62
+Subproject commit 71a4719e5f52f25857b51b3a4f2e781db5659f45
```

Comment:
I got this from Olly:

You probably will fix most things by doing a

'git submodule update'

Of course if you made and committed the changes in the submodule then you'll need to

'git add _submodule; git commit'